test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -356,18 +374,20 @@ def set_pbs_config(self, hostname=None, fin=None, fout=None,
                        to True
         :type append: boolean
         :param confs: The ``key/value`` pairs to create
-        :type confs: Dictionary
+        :type confs: Dictionary or None
         """
         if fin is None:
             fin = self.get_pbs_conf_file(hostname)
         if fout is None and fin is not None:
             fout = fin
-        if confs:
+        if confs is not None:
```

Comment:
`{}` is also not equal to `None`. Is this what you want?

Reply:
It is what I want.  I changed the default value of confs from {} to None because dictionaries are mutable types.  Originally, if confs was defaulted and it was changed, that change would persist through all calls to set_pbs_config().  By defaulting it to None and then setting it to an empty dictionary in the code, you avoid this.